extensions/sd_api_pictures/script.py

```diff
@@ -119,16 +136,16 @@ def output_modifier(string):
     if string == '':
         string = 'no viable description in reply, try regenerating'
 
-    # I can't for the love of all that's holy get the name from shared.gradio['name1'], so for now it will be like this
-    text = f'*Description: "{string}"*'
-
-    image = get_SD_pictures(string)
+    text = ""
+    if (params['mode']<2):
+        toggle_generation(False)
+        text = f'*Sends a picture which portrays: “{string}”*'
+    else:
```

Comment:
Perhaps `toggle_generation(True)` is missing here, as after switching to Aventure mode images does not automatically start to generate